blocks/query/block.json

```diff
@@ -99,21 +99,46 @@
     "orderby": {
       "default": "date",
       "enum": [
+        "ID",
+        "author",
+        "title",
+        "name",
+        "type",
         "date",
+        "modified",
+        "parent",
+        "rand",
+        "comment_count",
+        "relevance",
+        "menu_order",
+        "meta_value",
+        "meta_value_num",
```

Comment:
generally speaking, ordering by meta is going to slow the query down (potentially by A LOT on sites that don't have a meta_value index like sites on VIP do), but `meta_value_num` can be slow even on sites with an index. how slow depends on cardinality, so I can't speak in absolutes here, but given that this is a general use plugin I think including this by default is a bad idea.